IFC4x3/Properties/m/MinimumStorageTemperature_0EEQzV6qrBOw1KDdYos7$D/Documentation.md

```diff
@@ -0,0 +1 @@
+Minimum storage temperature that system complies with (C°)
```

Comment:
```suggestion
Minimum storage temperature that the system complies with (C°).
```

Reply:
or maybe a bounded value for `StorageTemperatureRange` with:

> Effective storage temperature range that the system or equipment can operate effectively at.